### DISCUSSION TITLE
posts/2023/08/record-feeling/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#21] [General]
# posts/2023/08/record-feeling/

https://guozheng.rbind.io/posts/2023/08/record-feeling/

**Comment by liechi** [comment]
我以前对记录不太感兴趣，因为在我看来，值得记住的事情是不会忘的；那些会忘掉的，就忘掉吧。不过，现在有点不确定了。因为发现即使事情能记住，但附在那些事情上的感受和情绪会逐渐淡忘---而那是最珍贵的。

所以现在折了个中：如果过一段时间后，一些人和事还会反复想起，那就把他们记下来---虽然不一定写在公开的地方。这样既不会太流水，也不会担心以后会忘记。

成长一般都有个追随的阶段，边追边找自己想去的方向。

**Comment by residualsun1** [comment]
> 因为发现即使事情能记住，但附在那些事情上的感受和情绪会逐渐淡忘---而那是最珍贵的。

这句话说到了我的心里。就我而言，我担心自己写作的能力，因为情绪起来的那一刻想到的话语可能是后来无法复现的，会被新的文字歪曲意思。

> 成长一般都有个追随的阶段，边追边找自己想去的方向。

感觉最近陷入了这个问题，内心有点无措。

**Reply by liechi** [reply]
我小学三年级从农村转学到城市里，远离了熟悉的环境和朋友。当时的城市在我心里神秘且高大上，自己作为一个没有见过世面又敏感的农村孩子，做什么都怕和别人不同，害怕被嘲笑。于是总默默观察城里的同学们是怎么说话怎么做事的，然后模仿他们，免得让自己显得格格不入。后来，我慢慢地发现身边的朋友也在模仿我说话语气和一些习惯动作。我才意识到，原来我在他们心中是很优秀的，他们也在观察我，并被我影响。慢慢建立起了自信心，开始习惯不用观察别人，而是按自己的所想去做事，去表达。

虽然这和你说的情况有些不同，但心理发展上或有些相似。我想说的是，模仿是学习的重要手段之一，不必因为模仿就觉得自己处于某种阴影里；对自己也不要妄自菲薄，因为在别人心里，包括你模仿的一些人的眼里，你或许也是阳光。

**Reply by residualsun1** [reply]
谢谢你，列驰。今晚在和小杨聊天的时候，我感觉我和她好像进入到了某种心流的状态。我和她每次聊天都会这样，或许是因为我们都对人类学有一定的好奇，她会很认真也很耐心地听我表达自己的看法，我也能感受出聊得有来有回的状态。虽然我只是一个门外汉，但是一些不成熟的看法反而让她觉得新奇，聊到后面，她捂着嘴，说：

> “和你一起交流，我学到了很多东西。”

不是恭维，也不是应和，我能感受得到是真心的“交换”第一次有了明确的回应，这给了我很大的鼓舞。

> 我想说的是，模仿是学习的重要手段之一，不必因为模仿就觉得自己处于某种阴影里；对自己也不要妄自菲薄，因为在别人心里，包括你模仿的一些人的眼里，你或许也是阳光。

前段时间方静也来了硇洲岛，和她夜聊的时候，她打破了我对她神秘、高大的幻想，告诉我我对他们的追逐（包括我说到自己从遇到兵哥以来感觉之后的每一处总有他的影子）都在给自己定下一个虚无的、无法抵达的标准。或许我们都是光，但在试着靠近别人的时候，却忘了自己也可以发光，或者正在发光。

当时心里是有某块裂痕被轻轻敲开的感觉——应该是从她指出我生活中处处是兵哥的影子——但不应该这样，虽然正因为兵哥，她才走出了自己的路，也走出了自己的精彩，走出了让我现在一度在不自觉追逐的精彩，可是自己要记得走回自己的路。

虽然我们当时好像在说一些听起来有点“悬”的话，我仍然能接收到她相信我在寻找自己的过程的鼓舞。

你们都走在我前头，给我的回应却都殊途同归。虽然是看似简单的几段话，但也让我想了很多。这几天我开始想到“自信”——虽然是小学就开始学习的概念，但多少成年人其实都没有好好感受过它。另，有时候过度地自我“反思”反而会让人陷入自我废弃——我将这理解为一种病态的自恋；虽然还会把自己的生活经历和别人的做比较，甚至往他者的标准靠近，但也会频频意识到自己在这么做，并试图去想想这自己的经历独特的地方在哪里。可能会有点摆烂，但不妨去想，这就是我，和别人不同的时候是我；尚且笨拙一点幼稚一点的文字是我；还在找自己的方向虽然还遥遥无期的是我。

感谢你从自己的经验真心地给予我思考的拐杖，用你的经历去尝试感受我当下的无措。
